test/sqllogictest/funcs.slt

```diff
@@ -931,5 +931,92 @@ SELECT tanh('-inf'::double)
 ----
 -1.000
 
+# Use the more reasonable number representation, as the standard mode
+# causes all sorts of weird representation issues with exp and log:
+mode cockroach
+
 query error function cot is only defined for finite arguments
 SELECT cot('-inf'::double)
+
+query error function log10 is not defined for zero
+SELECT log10(0.0::double)
+
+query error function log10 is not defined for zero
+SELECT log10(+0.0::double)
+
+query error function log10 is not defined for negative numbers
+SELECT log10(-1.0::double)
+
+query R
+SELECT log10(10.0::double)
+----
+1
+
+query R
+SELECT log(10.0::decimal(15, 5))
+----
+1
+
+query error function log is not defined for zero
+SELECT log(10, 0.0)
+
+query error function log is not defined for zero
+SELECT log(10, +0.0)
+
+query error function log is not defined for negative numbers
+SELECT log(10.0, -10)
+
+query R
+SELECT log(10, 10)
+----
+1
+
+query R
+SELECT log(400, 20)
+----
+2
+
+query R
+SELECT log10(10::decimal(15, 5))
+----
+1
+
+query R
+SELECT round(ln(13::float)::decimal(15, 5), 3)
```

Comment:
can we get tests for zeroes and negative numbers for both of the inputs here, even if they're expected to error?